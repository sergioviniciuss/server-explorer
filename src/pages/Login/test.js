import React from "react";
import { render } from "@testing-library/react";
import { Login } from ".";

describe("Login", () => {
  it("renders component properly", () => {
    const { container } = render(<Login />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it("renders logo in the header", () => {
    const { getByRole } = render(<Login />);

    expect(getByRole("img")).toBeInTheDocument();
  });
});
