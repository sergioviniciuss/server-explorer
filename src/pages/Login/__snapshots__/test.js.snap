// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders component properly 1`] = `
<div
  class="container"
>
  <img
    class="logo"
    src="/file"
  />
  <div
    class="wrapper"
  >
    <form
      class="container"
    >
      <input
        autocomplete="new-password"
        class="textField"
        placeholder="username"
        type="text"
        value=""
      />
      <input
        autocomplete="new-password"
        class="textField"
        placeholder="password"
        type="password"
        value=""
      />
      <button
        class="cta"
        type="submit"
      >
        Login
      </button>
    </form>
  </div>
</div>
`;
